project/app/utils/visualizations/histogram.py

```diff
@@ -0,0 +1,80 @@
+# Function to produce a histogram that shows the distribution of the scores
+# from the student's grade level for the current week
+# Plots a vertical line of the students most recent score for comparison
+
+# Imports
+import plotly.express as px
+
+
+def histogram(grade_list, student_info):
+    '''
+    Plotly histogram of all of the submission scores for the student's grade
+    for the current week
+    Plots a vertical line for the student's most recent score so they can see
```

Comment:
For [PEP8 compliance ](https://www.python.org/dev/peps/pep-0257/), there should be a blank line after the first summary line in a doc string.